### ISSUE TITLE
Cannot add a plug in settings

### ISSUE THREAD

**Issue by flak10monkey** [jneilliii/OctoPrint-Tasmota#139]
When I press the + icon to add a plug, nothing happens besides selecting the + icon with a square. Have tried selecting with mouse click and spacebar press.
Installed from Plugin Manager ver 1.0.2
Regular version of Octoprint venv
![shot1](https://user-images.githubusercontent.com/68299746/114255265-f4145d80-9979-11eb-8893-11039466adaa.png)

**Comment by jneilliii** [comment]
thanks @flak10monkey I know what's causing that. Expect a fix shortly.

**Comment by jneilliii** [comment]
Issue fixed in version 1.0.3 released just now.

**Comment by bereldhuin** [comment]
Hi,

I have the same issue, using 1.1.1 with Octoprint 1.7.3 (using Brave 1.37.116).
Looking in the browser console, I have the following error : 

```
5packed_libs.js?1fe9f29c:132 Uncaught ReferenceError: Unable to process binding "checked: function(){return is_sensor_only }"
Message: is_sensor_only is not defined
    at checked (eval at parseBindingsString (packed_libs.js?1fe9f29c:131:1741), <anonymous>:3:60)
    at e (packed_libs.js?1fe9f29c:131:2216)
    at packed_libs.js?1fe9f29c:131:3641
    at init (packed_libs.js?1fe9f29c:133:3412)
    at packed_libs.js?1fe9f29c:132:94
    at Object.G (packed_libs.js?1fe9f29c:130:11201)
    at packed_libs.js?1fe9f29c:132:75
    at Object.D (packed_libs.js?1fe9f29c:129:2126)
    at p (packed_libs.js?1fe9f29c:131:4001)
    at k (packed_libs.js?1fe9f29c:131:2747)
```

**Comment by jneilliii** [comment]
Ah, thanks for reporting. This is an issue with the latest update and the new sensor only option.

**Comment by jneilliii** [comment]
fixed with version 1.1.2.
